src/Signature/SignatureV4.php

```diff
@@ -82,7 +82,9 @@ public function presign(
         $expires,
         array $options = []
     ) {
-        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time']) : time();
+
+        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time'], null) : time();
```

Comment:
Similar to my other comment - how would you like the ternary broken up.  If you would prefer I could also switch it to a if/else which would have a shorter line length.